screens/AnnouncementScreen.js

```diff
@@ -41,6 +43,12 @@ class AnnouncementScreen extends React.Component {
     };
   }
 
+  async componentDidMount() {
```

Comment:
This doesn't need to be `async`, I think it breaks if you do?

Reply:
You right